lib/RudderClientV2.dart

```diff
@@ -0,0 +1,139 @@
+import 'package:flutter/services.dart';
+import 'package:rudder_sdk_flutter/RudderConfig.dart';
+import 'package:rudder_sdk_flutter/RudderOption.dart';
+import 'package:rudder_sdk_flutter/RudderProperty.dart';
+import 'package:rudder_sdk_flutter/RudderTraits.dart';
+
+class RudderClientV2 {
```

Comment:
Can you rename the class name? Something like `RudderController` or something like that?